cli/hzn.go

```diff
@@ -574,6 +574,7 @@ Environment Variables:
 	mmsObjectListDestinationId := mmsObjectListCmd.Flag("destinationId", msgPrinter.Sprintf("List mms objects with given destination id. Must specify --destinationType to use this flag")).String()
 	mmsObjectListWithData := mmsObjectListCmd.Flag("data", msgPrinter.Sprintf("Specify true to show objects that have data. Specify false to show objects that have no data. If this flag is omitted, both kinds of objects are shown.")).String()
 	mmsObjectListExpirationTime := mmsObjectListCmd.Flag("expirationTime", msgPrinter.Sprintf("List mms objects that expired before the given time. The time value is spefified in RFC3339 format: yyyy-MM-ddTHH:mm:ssZ. Specify now to show objects that are currently expired.")).Short('e').String()
+	mmsObjectListDeleted := mmsObjectListCmd.Flag("deleted", msgPrinter.Sprintf("Specify true to show objects that are marked deleted. Specify false to show objects that are not marked deleted. If this flag is omitted, both kinds of objects are shown. Object will be marked deleted if object is deleted in CSS but it doesn't receive notifications from all the destinations")).String()
```

Comment:
Let's reword, is this all true?
Specify true to show only deleted objects. Specify false to omit deleted objects. If this flag is omitted, both kinds of objects are shown. The data of a deleted objects no longer exists in the MMS, but a record of the object is retained until all known nodes have been notified that the object is deleted. To remove the record of a deleted object, reconnect all nodes to the network or delete the object a second time.

Reply:
These sentences look good to me. Thx!